provider/export/Cargo.toml

```diff
@@ -54,7 +54,7 @@ icu_provider_source = { path = "../source" }
 
 [features]
 default = ["rayon", "fs_exporter", "blob_exporter", "baked_exporter"]
-baked_exporter = ["dep:icu_provider_baked"]
+baked_exporter = ["dep:icu_provider_baked", "icu_locale/compiled_data"]
```

Comment:
compiled data should not be required

Reply:
fallbacker needs compiled data iirc

Reply:
the fallbacker is not created by this crate